src/Compiler/Scheme/ChezSep.idr

```diff
@@ -39,59 +39,67 @@ import Libraries.Utils.String
 %default covering
 
 schHeader : List String -> List String -> String
-schHeader libs compilationUnits = unlines
-  [ "(import (chezscheme) (support) "
-      ++ unwords ["(" ++ cu ++ ")" | cu <- compilationUnits]
-      ++ ")"
-  , "(case (machine-type)"
-  , "  [(i3le ti3le a6le ta6le tarm64le) (load-shared-object \"libc.so.6\")]"
-  , "  [(i3osx ti3osx a6osx ta6osx tarm64osx) (load-shared-object \"libc.dylib\")]"
-  , "  [(i3nt ti3nt a6nt ta6nt) (load-shared-object \"msvcrt.dll\")"
-  , "                           (load-shared-object \"ws2_32.dll\")]"
-  , "  [else (load-shared-object \"libc.so\")]"
-  , unlines ["  (load-shared-object \"" ++ escapeStringChez lib ++ "\")" | lib <- libs]
-  , ")"
-  ]
+schHeader libs compilationUnits = """
+  (import (chezscheme) (support)
+      \{ unwords ["(" ++ cu ++ ")" | cu <- compilationUnits] })
+  (case (machine-type)
+    [(i3le ti3le a6le ta6le tarm64le) (load-shared-object "libc.so.6")]
+    [(i3osx ti3osx a6osx ta6osx tarm64osx) (load-shared-object "libc.dylib")]
+    [(i3nt ti3nt a6nt ta6nt) (load-shared-object "msvcrt.dll")]
```

Comment:
Missing `(load-shared-object \"ws2_32.dll\")`

Reply:
I removed this line due to the removal of the same line in `Chez.idr`.